fleet-supervisor/src/utils.js

```diff
@@ -0,0 +1,20 @@
+const { execSync } = require('child_process')
+
+module.exports.getIPAddress = () => {
+  let address = '127.0.0.1'
+
+  try {
+    let command =`curl -s -X GET --header "Content-Type:application/json" "$BALENA_SUPERVISOR_ADDRESS/v1/device?apikey=$BALENA_SUPERVISOR_API_KEY"`
```

Comment:
Is this the best way to get the local IP? Thinking we can do it without an API request, but maybe this way is cleaner.

Reply:
I was less satisfied with the other alternatives I tried:
- ifconfig or route | grep 
- npm ip modules

If you have any suggestions happy to try them.

Reply:
@chrisys let me know if you have a suggestion otherwise I'll mark this as resolved.

Reply:
@tmigone happy to go with your preferred choice if the alternatives are no good